[Testing] Create a section dedicated to sending multiple requests in one functional test (#18789)
symfony/symfony-docs: testing.rst

    :ref:`framework.test <reference-framework-test>` option is enabled).
    This means you can override the service entirely if you need to.

Multiple requests in one test
.............................

After you send one request, subsequent ones will make the client reboot
the kernel, recreating the container from scratch.
This ensures that every requests are "isolated" using "new" service objects,
but can cause some unexpected behaviors. For example, the security token will
be cleared, Doctrine entities will be "detached"…

Calling the client's

> :method:`disableReboot() <Symfony\\Bundle\\FrameworkBundle\\KernelBrowser::disableReboot>`

Reviewer: ```suggestion
:method:`Symfony\Bundle\FrameworkBundle\KernelBrowser::disableReboot`
```

method is the first step to work around this, as this will reset the kernel
instead of rebooting it. Now, resetting the kernel will call the reset method
of every ``kernel.reset`` tagged service, which will **also** clear the

> security token, detach entities…

Reviewer: ```suggestion
security token, detach entities and so on.
```

Maybe ?

Author: Maybe ¯\_(ツ)_/¯

    :ref:`framework.test <reference-framework-test>` option is enabled).
    This means you can override the service entirely if you need to.

> Multiple requests in one test

Reviewer 2: ```suggestion
Multiple Requests in One Test
```

.............................

After you send one request, subsequent ones will make the client reboot
the kernel, recreating the container from scratch.

> This ensures that requests are "isolated" using "new" service objects,
> but can cause some unexpected behaviors. For example, the security token will

Reviewer: ```suggestion
This ensures that requests are "isolated" using "new" service objects.
However, this can cause some unexpected behaviors. For example, the security token will
```

be cleared, Doctrine entities will be "detached"…

Calling the client's
:method:`Symfony\\Bundle\\FrameworkBundle\\KernelBrowser::disableReboot`
method is the first step to work around this, as this will reset the kernel

> instead of rebooting it. Now, resetting the kernel will call the reset method

Reviewer: ```suggestion
instead of rebooting it. Now, resetting the kernel will call the ``reset()`` method
```

of every ``kernel.reset`` tagged service, which will **also** clear the
security token, detach entities and so on.

> As such, the next step is to create a

Reviewer: ```suggestion
As such, a necessary step is to create a
```

Using "next step" feels weird to me as there is no other step in this section

Author: The previous step is the previous paragraph (and is mentioned as such); is that not clear enough? :thinking:

Reviewer: Ah yes that's right, my bad 👍
